oaknational/Oak-Web-Application
# [CUR-180]: added video guide if video data is being passed through to curriculum overview

## Description

- Update gql query to account for video information if being passed through
- Added video section to overview page

## How to test

1. Go to https://deploy-preview-1969--oak-web-application.netlify.thenational.academy/teachers/curriculum/english-primary/overview
2. Scroll down to the video section
3. You should see a placeholder video
4. Go to any other subject phase combination, the video guide shouldn't be present

## Screenshots

How it used to look (delete if n/a):
<img width="1379" alt="Screenshot 2023-10-04 at 12 06 57" src="https://github.com/oaknational/Oak-Web-Application/assets/32605982/1bdfa12b-0020-4d79-a35f-11df7ea6bda4">

How it should now look:
<img width="1351" alt="Screenshot 2023-10-04 at 14 28 27" src="https://github.com/oaknational/Oak-Web-Application/assets/32605982/6f2f740f-e36f-467b-a622-63a071dea3fc">


## Checklist
